docs/usage/config-presets.md

```diff
@@ -111,27 +111,27 @@ It mostly uses Renovate config defaults but adds a few smart customizations such
 
 ## How to Use Preset Configs
 
-By default, Renovate App's onboarding PR suggests the `["config:base]"` preset.
-If you're self hosting, and want to use the `config:base` preset, then you must add `"onboardingConfig": { "extends": ["config:base"] }` to your bot's config.
+By default, Renovate App's onboarding PR suggests the `["config:recommended]"` preset.
+If you're self hosting, and want to use the `config:recommended` preset, then you must add `"onboardingConfig": { "extends": ["config:recommended"] }` to your bot's config.
```

Comment:
Maybe it's time to make this the onboardingConfig too?

Reply:
yes, did I miss that anywhere?

Reply:
Is `config:recommended` now the onboarding configuration?